hack/build.sh

```diff
@@ -41,11 +41,6 @@ if [[ -z "${VERSION}" ]]; then
     exit 1
 fi
 
-if [[ -z "${GIT_SHA}" ]]; then
```

Comment:
`GIT_SHA` was required to set the version fields in the `buildinfo` struct.
Now by removing this check, we are making this optional and setting the `buildinfo` field only if it is supplied.
Which IMO is not correct.
Changes to this file can be reverted.

Reply:
Reverted ✅